nengo_gui/components/htmlview.py

```diff
@@ -10,31 +12,50 @@ class HTMLView(Component):
 
     def __init__(self, obj):
         super(HTMLView, self).__init__()
-        self.obj = obj
-        self.obj_output = obj.output
         self.data = collections.deque()
 
+        self.obj = obj
+        self.obj_output = self.obj.output
```

Comment:
Is there a difference between `self.obj_output` and `self.obj.output`? There's one line `self.obj.output = self.obj_output` in a function which is doing the opposite assignment and I don't understand why. The two variables don't seem to be kept in sync and it seems to be random which one is used where.